src/seed/seed_db.ts

```diff
@@ -708,29 +715,37 @@ async function reloadAutocompleteData(): Promise<void> {
                 .orderBy("publishedon", "desc")
                 .execute();
 
+            const generateSongArtistIdentifier = (
+                x: AvailableSongDeltaData,
+            ): string => `${x.song_name_en},${x.artist_name_en}`;
+
             const availableSongsAfterSet = new Set(
-                availableSongsAfter.map((x) => x.link),
+                availableSongsAfter.map(
+                    (x) => `${generateSongArtistIdentifier(x)}`,
```

Comment:
The string conversion here and below can be removed

Reply:
im assuming u mean the template string